custom_components/garmin_connect/__init__.py

```
@@ -431,14 +441,13 @@ async def _async_update_data(self) -> dict:
                 raise ConfigEntryAuthFailed from err
             if err.response.status_code == 429:
                 _LOGGER.error(
-                    "Too many requests error occurred during update: %s", err.response.text)
+                    "Too many requests error occurred during update: %s", err.response.text
+                )
```

Ruff (0.14.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)